docs/docs/how-to/cloud/adding-a-custom-domain.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Adding a Custom Domain to Your Site on Gatsby Cloud
+description: Learn how to add a custom domain to your site on Gatsby Cloud
+---
+
+## Introduction
+
+Gatsby Cloud allows you to specify Custom Domains for your Gatsby websites. This guide provides the steps for applying Custom Domains to your sites on Gatsby Cloud.
+
+## Prerequisites
+
+Before you can add a custom domain, you'll need to be hosting with Gatsby Cloud. If you haven't already, [turn on Gatsby Hosting](https://support.gatsbyjs.com/hc/en-us/articles/360061641114).
+
+## Directions
+
+### 1. Add a Domain
```

Comment:
The title casing is still incorrect on most headings throughout the docs

Reply:
[Section titles](https://www.gatsbyjs.com/contributing/gatsby-style-guide/#format-titles-and-headers) are to follow sentence casing, which they all do now.